assets/src/stories-editor/components/context-menu/index.js

```diff
@@ -201,57 +207,58 @@ const ContextMenu = ( props ) => {
 			},
 
 		];
+	}
+
+	// Disable Duplicate Block option for cta and attachment blocks.
+	if ( block && ! DISABLE_DUPLICATE_BLOCKS.includes( block.name ) ) {
```

Comment:
This check for `if ( block` is used sporadically at best. Isn't there always a `block`?

Reply:
You are right. Fixed in https://github.com/ampproject/amp-wp/pull/3537/commits/b40e5b6efbc7d0e91ceb9fe8c8080d4f4d7c7ff3

Reply:
You added in back in because it was necessary for E2E, or? Was there a case where `block` is `null`? Can't we just abort early in this entire function if that's the case?

Reply:
There are cases where block is not set. I saw it in my tests, I was setting a warning. This check for block, is not part of this PR, it was in before. 

https://github.com/ampproject/amp-wp/blob/develop/assets/src/stories-editor/components/context-menu/index.js#L205